sklearn/utils/tests/test_utils.py

```diff
@@ -674,10 +674,14 @@ def test_print_elapsed_time(message, expected, capsys, monkeypatch):
         ("", False),
         ("nan", False),
         ([np.nan], False),
+        (9867966753463435747313673, False),  # Python int that overflows with C type
+        (9867966753463435747313673.0, False),  # Python int that overflows with C type
```

Comment:
```suggestion
        (9867966753463435747313673.0, False),  # Python float that overflows with C type
```
Does this float really overflow or just skip digits at the end? Did it raise an error before?

Reply:
Yep this case does not overflow. It does not make that much sense because it will be changed to `inf`